src/server/visual-diff-reporter.js

```diff
@@ -52,7 +63,6 @@ function flattenResults(session, browserData, fileData) {
 			const testData = fileData.tests.get(testName);
 			if (!t.passed) {
 				browserData.numFailed++;
-				fileData.numFailed++;
```

Comment:
Counting it here caused it to think that a test failed 3 times if it failed in all 3 browsers, which isn't quite what I want to surface.